librz/core/p/core_java.c

```diff
@@ -1935,15 +1605,72 @@ static bool rz_cmd_java_handle_print_exceptions(RzCore *core, const char *input)
 			free(class_info);
 		}
 	}
-	return true;
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_cmd_java_handler(RzCore *core, int argc, const char **argv) {
+	if (argc < 2) {
```

Comment:
since you have made the `subcmd` argument optional, you should handle it here.

Reply:
when you exec `java` newshell will only provide `argv[][]= {{"java"}}` and argc is `1`.